trie/statistics/trieStatisticsCollector_test.go

```diff
@@ -53,3 +54,17 @@ func getTrieStats(maxLevel int, size uint64) common.TrieStatisticsHandler {
 
 	return ts
 }
+
+func TestGetNumTriesByTypeString(t *testing.T) {
```

Comment:
this test will seldom fail. Because the iteration on trie/statistics/trieStatisticsCollector.go L106 is done on a map.